generator/config/accumulator/lastN.yaml

```diff
@@ -2,10 +2,13 @@
 name: $lastN
 link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/lastN/'
 type:
+    - accumulator
     - window
```

Comment:
For [`$addToSet`](https://www.mongodb.com/docs/manual/reference/operator/aggregation/addToSet/#mongodb-group-grp.-addToSet), I see the following clearly mentioned at the top of the page in the list of places where the operator is supported:

> * [$setWindowFields](https://www.mongodb.com/docs/manual/reference/operator/aggregation/setWindowFields/#mongodb-pipeline-pipe.-setWindowFields) (Available starting in MongoDB 5.0)

For [`$lastN`](https://www.mongodb.com/docs/manual/reference/operator/aggregation/lastN/#window-function-and-aggregation-expression-support), the same information is buried much further down the page.

This seems very inconvenient.

----

Do all of the window operators have identical syntax to the non-window accumulator? I assume that's also why these configs appear under an `accumulator/` directory instead of being duplicated into a `window/` directory (as you had to do with `expression/`).

Reply:
Window Operators are a subset of Group Accumulators. That's why I didn't create a separate category.